be2-scala/src/main/scala/ch/epfl/pop/model/network/method/message/data/socialMedia/AddBroadcastChirp.scala

```diff
@@ -0,0 +1,28 @@
+package ch.epfl.pop.model.network.method.message.data.socialMedia
+
+import ch.epfl.pop.json.MessageDataProtocol._
+import ch.epfl.pop.model.network.Parsable
+import ch.epfl.pop.model.network.method.message.data.ActionType.ActionType
+import ch.epfl.pop.model.network.method.message.data.ObjectType.ObjectType
+import ch.epfl.pop.model.network.method.message.data.{ActionType, MessageData, ObjectType}
+import ch.epfl.pop.model.objects.{Hash, Timestamp}
+import spray.json._
+
+case class AddBroadcastChirp(
+                          chirpId: Hash,
+                          channel: String,
```

Comment:
Is there a reason why you did not define it as a `Channel`?

Reply:
Actually no, I'll change that

Reply:
Actually yes, to make it easier, as for now there is no converter from Channel to Json